tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -1072,6 +1072,13 @@ platform_tests/test_advanced_reboot.py:
       - "asic_type in ['vs']"
       - "release in ['202412']"
 
+platform_tests/test_advanced_reboot.py::test_fast_reboot:
+  skip:
+    reason: "Skip for smartswitch topology"
+    conditions_logical_operator: or
```

Comment:
There is only one condition, we can remove `conditions_logical_operator`

Reply:
Hi @ZhaohuiS  Updated